config/cable.yml

```
@@ -7,4 +7,4 @@ test:
 production:
   adapter: redis
   url: redis://localhost:6379/1
-  channel_prefix: caseflow_efolder_production
+  channel_prefix: caseflow_efolder_production
```

Looks like the newline got accidentally removed here:

```suggestion
  channel_prefix: caseflow_efolder_production

```